### DISCUSSION TITLE
GAT

### DISCUSSION THREAD

**Discussion by shubhiambast** [pyg-team/pytorch_geometric#9322] [Q&A]
Hi, I want to use graph attention networks comparing both transdcutive and inductive settings. As per my knowledge, loaders from torch_geometric.loader are used to efficiently sample and scale up the training process. Which loaders are compatible for graph attention networks ? Can I use cluster-GCN or NeighborLoader for GAT Or I'm mistaken ?

**Comment by wsad1** [comment]
Yeah you can use `ClusterLoader` or `NeighborLoader` with `GAT`.
